src/unity/python/turicreate/test/test_sframe.py

```diff
@@ -204,11 +204,29 @@ def test_auto_parse_csv(self):
             df.to_csv(csvfile, index=False)
             csvfile.close()
 
-            sf = SFrame.read_csv(csvfile.name, header=True)
+            import codecs
+            with open(csvfile.name) as f:
+                content = f.read()
+            with open(csvfile.name, 'w') as f:
+                f.write(codecs.BOM_UTF8)
+                f.write(content)
 
+            sf = SFrame.read_csv(csvfile.name, header=True)
             self.assertEqual(sf.dtype, [float, int, str])
             self.__test_equal(sf, df)
 
+    def test_csv_with_BOM(self):
```

Comment:
It looks like the test names got switched; `test_csv_with_BOM` tests without a BOM, and `test_auto_parse_csv` tests with a BOM now. Was that intended?

Reply:
Oh. That was a mistake. Fixing